routes/route_team.jsx

```diff
@@ -83,6 +83,10 @@ const WAKEUP_CHECK_INTERVAL = 30000; // 30 seconds
 const WAKEUP_THRESHOLD = 60000; // 60 seconds
 
 function preNeedsTeam(nextState, replace, callback) {
+    if (nextState.location.action === 'REPLACE') {
+        callback();
+        return;
+    }
     if (RouteUtils.checkIfMFARequired(nextState)) {
```

Comment:
@jwilander Does this block need to be moved before the replace action? I think this order should be fine since turning on MFA enforcement would force a refresh, but I just want to make sure

Reply:
Yeah it shouldn't be a problem because of the refresh